src/test/test-Transform.ts

```diff
@@ -347,7 +347,7 @@ describe("Transform", () => {
 					await delay(2);
 					await s.write(2);
 				} finally {
-					await s.end();
+					await s.end(undefined, Promise.resolve());
```

Comment:
It's probably wise to add the comment that I added before this line in one of my comments, to tell people why you pass in `Promise.resolve()`. I.e. that it's *not* normal to do so, and only happens here because we want to explicitly test for it. Same for the case a few lines down.